cmd/vic-machine/configure/configure.go

```diff
@@ -169,15 +171,21 @@ func (c *Configure) copyChangedConf(o *config.VirtualContainerHostConfigSpec, n
 		hProxy := ""
 		if c.HTTPProxy != nil {
 			hProxy = c.HTTPProxy.String()
+			updateSessionEnv(personaSession, config.GeneralHTTPProxy, hProxy)
```

Comment:
This restructuring fixes one issue as noted in the commit message, but looks like it may introduce another: it no longer seems possible to unset a previously-configured proxy.

Reply:
Good catch. Fixed the regression.